pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +27,19 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+     
+     JSONArray ja = new JSONArray();
```

Comment:
Each call to `doGet` would add another comment to this array. Is that what is supposed to happen here?

Reply:
Yeah that was how I was testing it but there will be a post request, this is just for the get request of the comments. Now, with the correction I made, the comments array is declared and the test comment is added every time I do the get request to "/data" but it's just for testing purposes. :)